src/web_ui/__init__.py

```diff
@@ -1,27 +1,38 @@
 # Set up logging
 import logging
 
-from multiprocessing.sharedctypes import Value
+from constants import NAME_KEY_MAP
```

Comment:
We'll have to obtain this "map" based on my comment on the ticket.

Reply:
This was removed in a later commit already, I'm not sure why it's still here